api/python/t4/packages.py

```diff
@@ -85,13 +86,13 @@ def read_physical_key(physical_key):
         raise NotImplementedError
 
 
-def get_package_registry(path):
-    """ Returns the package registry for a given path """
+def get_package_registry(path=''):
+    """ Returns the package registry root for a given path """
```

Comment:
Looks like it returns a string in one case and a path in the other, can you update? probably has to be a file uri string for the local package instead of a Path object